pkg/repository/v1_repository.go

```diff
@@ -639,8 +646,8 @@ func (r *V1Repository) UpdateComponentManifests() error {
 	}
 
 	for name := range index.Components {
-		_, err = r.updateComponentManifest(name)
-		if err != nil {
+		_, err = r.updateComponentManifest(name, false)
+		if err != nil && err != errUnknownComponent {
```

Comment:
It's better to unwrap the error to prevent the original error from being wrapped with stack or annotation.